Packs/Zoom/Integrations/Zoom/Zoom.py

```diff
@@ -582,61 +606,72 @@ def zoom_create_meeting_command(client, **args) -> CommandResults:
     )
 
 
-def zoom_fetch_recording_command():
-    # this is the original code with no changes at all.
-    # this part will be removed in the future
-    # waiting for a paid account
-    def get_jwt(apiKey, apiSecret):
-        """
-        Encode the JWT token given the api ket and secret
-        """
-        tt = datetime.now()
-        expire_time = int(tt.strftime('%s')) + 5000
-        payload = {
-            'iss': apiKey,
-            'exp': expire_time
-        }
-        encoded = jwt.encode(payload, apiSecret, algorithm='HS256')
-        return encoded
-    URL = 'https://api.zoom.us/v2/'
-    ACCESS_TOKEN = get_jwt(demisto.getParam('apiKey'), demisto.getParam('apiSecret'))
-    PARAMS = {'access_token': ACCESS_TOKEN}
-    HEADERS = {'Content-Type': 'application/json', 'Accept': 'application/json'}
-    USE_SSL = not demisto.params().get('insecure', False)
-    meeting = demisto.getArg('meeting_id')
-    res = requests.get(URL + 'meetings/%s/recordings' % meeting, headers=HEADERS, params=PARAMS, verify=USE_SSL)
-    if res.status_code == requests.codes.ok:
-        data = res.json()
-        recording_files = data['recording_files']
-        for file in recording_files:
-            download_url = file['download_url']
-            r = requests.get(download_url, stream=True)
-            if r.status_code < 200 or r.status_code > 299:
-                return_error('Unable to download recording for meeting %s: [%d] - %s' % (meeting, r.status_code, r.text))
-
-            filename = 'recording_%s_%s.mp4' % (meeting, file['id'])
+def zoom_fetch_recording_command(client: Client, **args):
+    # preprocessing
+    results = []
+    meeting_id = args.get('meeting_id')
+    delete_after = argToBoolean(args.get('delete_after'))
+    client = client
+
+    data = client.zoom_fetch_recording(
+        method='GET',
+        url_suffix=f'meetings/{meeting_id}/recordings'
+    )
+    recording_files = data.get('recording_files')
+    # getting the audio and video files
+    for file in recording_files:
```

Comment:
Is is possible to have no recording? If so, do we want to have a better notice to the client?

Reply:
that is possible. the error will be raised bi the first call to the API (L616)